doc/community/developer-guide.md

```diff
@@ -110,7 +110,7 @@ In our codebase, we manage connections to databases with a `Connection` object,
 
 ### Working with connections
 
-`Connection` should be exclusively used to manage database connections on the user's behalf and to obtain the current SQLAlchemy connection. We can access the current SQLAlchemy connection using `current.session`.
+`ConnectionManager` should be exclusively used to manage database connections on the user's behalf and to obtain the current connection. We can access the current connection using `current`.
```

Comment:
There's a line above this : `In our codebase, we manage connections to databases with a Connection object,` .. should this also be `ConnectionManager` ?